### PR TITLE
It had Bootstrap instead of Bootswatch in documentation

### PR DESCRIPTION
**Pull request by lesleyauk**
I also improved readability and reduced confusing repetition of the word "theme" here.

### CODE REVIEW

File: docs/user-guide/styling-your-docs.md [hunk 1]
```diff
@@ -30,12 +30,11 @@ the [custom themes](#custom-themes) section below.
 
 ![ReadTheDocs](https://docs.readthedocs.org/en/latest/_images/screen_mobile.png)
 
-## Bootstrap and Bootswatch Themes
+## Bootstrap and Bootswatch themes
```
Comment (waylan) [inline docs/user-guide/styling-your-docs.md]: This is a section header/subtitle so it should follow the [capitalization rules][1] of a title. In other words, "Themes" should be capitalized.

[1]: http://capitalizemytitle.com/title-capitalization-rules/

**Comment by waylan** [comment]
@lesleyauk thanks for your contribution. This is a good change in that it is correcting a clear error in the documentation.  However, given that #841 is proposing that that entire section be removed, this seems kind of moot. Given that there seems to be some question regarding whether #841 will be accepted, I won't close this just yet. Let's wait and see what happens there first.

File: docs/user-guide/styling-your-docs.md (hunk 1 above)
Comment (lesleyauk) [inline docs/user-guide/styling-your-docs.md]: @waylan There is some inconsistency in the documentation about whether to use title case or sentence case. I saw the page heading "Styling your docs" and the subheading above "Built-in themes" were sentence case so went with that. FWIW, I do happen to prefer sentence case but will bow to whatever you want to use... Not many people have an editorial style guide for this kind of thing in their repo :-)

Comment (waylan) [inline docs/user-guide/styling-your-docs.md]: I have to admit, I didn't look at the rest of the document before making my comment. If you are following the style of the rest of the document, let's leave your change in for now. If we want to change the style, we can do that separately. However, that is @d0ugal 's call.

Comment (d0ugal) [inline docs/user-guide/styling-your-docs.md]: LGTM. 👍

At some point we need to do a review of the documentation style, but we probably need a better writer (than me) involved in the project.

**Comment by d0ugal** [comment]
I'll merge this for now and rebase #841 on it.

File: docs/user-guide/styling-your-docs.md (hunk 1 above)
Comment (lesleyauk) [inline docs/user-guide/styling-your-docs.md]: @d0ugal I've only just started using MkDocs and have found most of the documentation to be good quality. Ping me if you want a proofreader any time!

Comment (d0ugal) [inline docs/user-guide/styling-your-docs.md]: @lesleyauk I'll hopefully take you up on that offer :smile: Please do open an issue if you have any other documentation feedback. Thanks!

**Pull request merged**